src/SourceBuild/content/repo-projects/scenario-tests.proj

```diff
@@ -4,54 +4,46 @@
     <!-- The scenario-tests repo shouldn't be cleaned after building as we run tests from it. -->
     <CleanWhileBuilding>false</CleanWhileBuilding>
 
-    <ScenarioTestsResultsDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsTestResultsDir)', 'scenario-tests'))</ScenarioTestsResultsDir>
-    
-    <TestArgs>$(TestArgs) /p:TestBinlogDir=$(ArtifactsLogRepoDir)</TestArgs>
-    <!-- Pass the path of the updated NuGet.config file in which contains the live source-built feeds. Skip this when building source-only
-         as in that configuration we currently don't want to use the live generated NuGet packages from the various repositories. -->
-    <TestArgs Condition="'$(NuGetConfigFile)' != '' and '$(DotNetBuildSourceOnly)' != 'true'">$(TestArgs) /p:RestoreConfigFile=$(NuGetConfigFile)</TestArgs>
+    <TestArgs>$(TestArgs) /p:TestBinlogDir="$(ArtifactsLogRepoDir.TrimEnd('\'))"</TestArgs>
```

Comment:
Do we need to cover both Windows and Linux cases with, for instance `TrimEnd('/\')`?